client/src/legacy/MemberImportForm.js

```diff
@@ -7,7 +7,7 @@ $.entwine('ss', function($){
   /**
    * Class: .import-form .advanced
    */
-  $('.import-form .advanced').entwine({
+  $('#Form_ImportForm .advanced').entwine({
```

Comment:
We shouldn't need this javascript at all - we should rely on what `ModelAdmin` already provides for showing hidden content instead of duplicating that functionality.

Reply:
Removed